configuration/app-settings.md

```diff
@@ -0,0 +1,17 @@
+# App settings
+
+Much of the customization lives in the app_settings. Look for the `app_settings` field in the design doc for `medic` db.
+
+![app_settings](img/app_settings.png)
+
+You can update these settings with the [scripts/update_settings.js](https://github.com/medic/medic-webapp/blob/master/scripts/update_app_settings.sh) script, or in Dashboard (`<host>/dashboard/_design/dashboard/_rewrite/`), or by editing the file in Futon directly.
+
```

Comment:
Lines 3 through 7 are out of date now - the app settings are stored in a separate doc, and you should use medic-configurer to update them, not a script. Maybe just replace this with...

```
Much of the configuration of the app is stored in the database in a document with id "settings". To update settings use the medic-conf command line tool.
```

... with a link to medic-conf: https://github.com/medic/medic-conf